operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -0,0 +1,257 @@
+        |||) +
+        panel.withPosition({ x: 12, y: 1, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_node_gossip_received_events_total{instance="$instance"}[$__rate_interval])',
+            legendFormat='{{event}}'
+          ),
+        ])
+      ),
+      // Known peers
+      (
+        panel.new('Known peers', 'stat') +
+        panel.withDescription(|||
+          Known peers to the node (including the local node).
+        |||) +
+        panel.withPosition({ x: 0, y: 9, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='sum(cluster_node_peers{instance="$instance"})',
+          ),
+        ]) +
+        panel.withUnit('suffix:peers')
+      ),
+      // Peers by state
+      (
+        panel.new('Peers by state', 'timeseries') +
+        panel.withDescription(|||
+          Known peers to the node by state.
+        |||) +
+        panel.withPosition({ x: 12, y: 9, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='cluster_node_peers{instance="$instance"}',
+            legendFormat='{{state}}',
+          ),
+        ]) +
+        panel.withUnit('suffix:nodes')
+      ),
+      // Gossip Transport row
+      (
+        panel.new('Gossip Transport', 'row') +
+        panel.withPosition({ h: 1, w: 24, x: 0, y: 17 })
+      ),
+      // Transport bandwidth
+      (
+        panel.new('Transport bandwidth', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 0,
+          y: 18,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_transport_rx_bytes_total{instance="$instance"}[$__rate_interval])'
+          ),
+          panel.newQuery(
+            expr='-1 * rate(cluster_transport_tx_bytes_total{instance="$instance"}[$__rate_interval])'
+          ),
+        ]) +
+        panel.withUnit('Bps')
+      ),
+      // Packet write success rate
+      (
+        panel.new('Packet write success rate', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 8,
+          y: 18,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_tx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Tx success %',
+          ),
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_rx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_rx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Rx success %',
+          ),
+        ]) +
+        panel.withUnit('percentunit')
+      ),
+      // Pending packet queue
+      (
+        panel.new('Pending packet queue', 'timeseries') +
```

Comment:
Can we add a description here so the user knows what the pending packet queue is, and when they should be concerned about it?

Reply:
How does this sound?

```
          The number of packets enqueued currently to be decoded or encoded and sent during communication with other nodes.

          The incoming and outgoing packet queue should be as empty as possible; a growing queue means that the Agent cannot keep up with the number of messages required to have all nodes informed of cluster changes, and the nodes may not converge in a timely fashion.
```

Reply:
Perfect 👍